dataLayer/modulemanager.datalayer.js

```diff
@@ -2,26 +2,36 @@ MMdataLayer = [{
     "languageIsoCode": "en",
     "countryIsoCode ": "US",
     "originLocations ": [
-      "MIA","TXL","SXF"
+        "MIA", "TXL", "SXF"
     ],
     "destinationLocations": [
-      "LHR","MIL","LAX"
+        "LHR", "MIL", "LAX"
     ],
     "productCategory": "ONE_WAY",
     "productType": "ECONOMY",
-    "loyaltyProgram":{
-      "isLoggedIn": true,
-      "membershipLevel": "Basic",
-      "redemptionUnit": "MILES",
-      "redemptionBalance": 2500,
-      "lastSearchedProducts": [
-        "MIA>JFK"
-      ],
-      "homeLocation": "MIA",
-      "savedProducts": [
-        "NYC>CHI","ORD>MIA"
-      ],
-    "preferredProductCategory": "ROUND_TRIP",
-    "preferredProductType": "BUSINESS"
+    "loyaltyProgram": {
+        "isLoggedIn": true,
+        "membershipLevel": "Basic",
+        "redemptionUnit": "MILES",
+        "redemptionBalance": 2500,
+        "lastSearchedProducts": [
+            "MIA>JFK"
+        ],
+        "homeLocation": "MIA",
+        "savedProducts": [
+            "NYC>CHI", "ORD>MIA"
+        ],
+        "preferredProductCategory": "ROUND_TRIP",
+        "preferredProductType": "BUSINESS"
+    },
+    "experience": {
+        "type": "Ticket",
+        "name": "Concert"
+    },
+    "event": {
+        "location": "Margaret Court Arena",
+        "date": "Day 1",
+        "session": "Night",
```

Comment:
Same for the next two variables. According to the context these should be arrays. Can you please check?